src/models/categoryDAO.js

```diff
@@ -1,29 +1,30 @@
 import prisma from '../../prisma';
 
-const getMainCategoryList = async () => {
-  const mainCategories = await prisma.$queryRaw`
+export const getMainCategories = async () => { // md추천 호캉스..
+  const getCategories =  await prisma.$queryRaw`
     SELECT
       mc.id,
-      mc.name as mainCategoryId,
+      mc.name as mainCategoryName
     FROM
-      main_categories
-  `
-}
+      main_categories mc
+  ;
+  `;
+  return getCategories
+};
 
 
-const getSubCategoryList = async (mainCategoryId) => {
-  const subCategories = await prisma.$queryRaw`
+export const getSubCategoryList = async () => { // 지역별
+  const getSubCategoryList = await prisma.$queryRaw`
```

Comment:
```suggestion
export const getSubCategoryList = async (mainCategoryId) => { // 지역별
  const getSubCategoryList = await prisma.$queryRaw`
```

where clause 걸어주세요 :)